package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "3.4.9",
+  "version": "3.5.0",
```

Comment:
If @shabeeb-aot 's PR is merged first then this is fine.

If my PR is merged first then Faiz will have to update his (perhaps go to 3.6.0?).